functions/system.bash

```diff
@@ -130,6 +130,7 @@ vimrc_copy() {
 srv_bind_mounts() {
   echo -n "$(timestamp) [openHABian] Preparing openHAB folder mounts under /srv/... "
   sed -i "\\#[ \\t]/srv/openhab2-#d" /etc/fstab
+  # dummy "
```

Comment:
?

Reply:
😆 vim coloring fails to work properly without that because it doesn't detect the opening " in the line before.
just forgot to remove. There's another PR coming up to remove that part anyway